src/Native/Windows/build-native.cmd

```diff
@@ -10,36 +10,53 @@ set __IntermediatesDir=""
 set __BuildArch=AnyCPU
 set __VCBuildArch=x86_amd64
 set CMAKE_BUILD_TYPE=Release
+set __EnvironmentSet=false
+set __VSVersion=vs2013
```

Comment:
I believe that CMake saves you from maintaining boring stuff: detection of where things are, reasonable defaults, ... . make would give the same build driver, but you pay for it by having more platform-specific plumbing that has be maintained.